.eslintrc.js

```diff
@@ -12,5 +12,6 @@ module.exports = {
     browser: true
   },
   rules: {
+    'ember-suave/no-direct-property-access': 'off'
```

Comment:
the new changes brought in by ember init are in disagreement with this rule. I'd rather we follow "the Ember way" than deal with line-noise in every upgrade's `ember init`